api/src/main/java/org/openmrs/module/pihcore/PihCoreActivator.java

```diff
@@ -111,6 +111,8 @@ public void started() {
             setGlobalProperties(config);
             setExtraIdentifierTypes(metadataMappingService, patientService, config);
             MergeActionsSetup.registerMergeActions();
+            // TODO line below will be removed when we migrate location tags
+            LocationTagSetup.setupLocationTagsLegacy(locationService, config);
             LocationTagSetup.setupLocationTags(locationService, config);
```

Comment:
We actually can start overriding what's defined in the "legacy" code with a config-based assignment immediately since this is called after the legacy.

It also occurs to me, fwiw, we can start migrating the location tag mapping prior to migrating the locations themselves to Iniz.  MIght be worth doing.